com.heroicgameslauncher.yml

```diff
@@ -0,0 +1,145 @@
+id: com.heroicgameslauncher.hgl
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+command: heroic-run
+separate-locales: false
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Compat.i386
+  - org.freedesktop.Sdk.Extension.toolchain-i386
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --allow=multiarch
+  - --allow=devel
+  - --device=all
```

Comment:
Is `all` required because of joysticks and controllers?

Reply:
Yes. It is a game launcher so we need at least Joysticks. 
I could not find a way of limiting this permission though.
Also, I believe that for devices like the Steam Deck that would be good to avoid issues.

Reply:
There's no way to limit these to certain device types, it would need new portals.